channels/ru.m3u

```diff
@@ -689,12 +685,8 @@ https://strm.yandex.ru/kal/start/start0.m3u8
 https://sc.id-tv.kz/STV.m3u8
 #EXTINF:-1 tvg-id="StranaFM.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Music",Страна FM (720p) [Offline]
 http://live.stranafm.cdnvideo.ru/stranafm/stranafm_hd.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="StrashnoeHD.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",Страшное HD (720p)
-http://spacetv.in/stream/DIPK8H19CP8/226.m3u8
```

Comment:
http://stream.backustv.ru:80/live/btv2/index.m3u8

Reply:
Added in #5501